Patch by <kees@ubuntu.com> for dsniff >= 2.4b1, which includes a missing
header file to make it building.

--- dsniff-2.4b1/ssh.c			2006-10-12 13:21:57.000000000 -0700
+++ dsniff-2.4b1/ssh.c.openssl_098	2006-10-12 13:22:46.441893077 -0700
@@ -16,6 +16,7 @@
 #include <openssl/ssl.h>
 #include <openssl/err.h>
 #include <openssl/rand.h>
+#include <openssl/md5.h>
 
 #include <err.h>
 #include <errno.h>
